### DISCUSSION TITLE
Preselecting genes based on known involvement

### DISCUSSION THREAD

**Discussion by AngCamp** [theislab/scvelo#924] [General]
Hi there I am interested in a biological process that lasts around 8 hours (which I hope will be acceptable for splicing dynamics assumption made by velocyto or scVelo). There are around 400 genes which are known in the cell type in question to be DEGs of the process.  These genes have been identified in settings where it was possible to enrich for cell engaged in the process  I am looking in publicly available datasets for their occurrence, partially because I am interested in the "natural abundance" of this process under different conditions.

The cells should be fairly rare so I was wondering if it would be valid to only look at the RNA velocity of the genes known to be involved in the process to allow fitting of the model on genes of interest rather than computing a velocity across all genes?  I'm curious if doing so would enhance the signal or if it would be breaking any assumptions made by velocyto or scVelo.

This would have to occur after the preselecting steps of course: https://github.com/theislab/scvelo/discussions/686

**Comment by WeilerP** [comment]
@AngCamp, selecting the genes would not break any assumptions. However, it is not guaranteed that the genes exhibit the necessary dynamics simply because they are important for the process (see e.g. https://github.com/theislab/scvelo/issues/216 or https://github.com/theislab/scvelo/discussions/462 for potential issues). You could use, our recently proposed permutation score (see [here](https://www.biorxiv.org/content/10.1101/2022.08.12.503709v1)), however, to more easily check if the genes exhibit transient cell types. ATM, veloVI is still a separate package but will be integrated into scVelo in the near future.
